src/_scss/pages/covid19/index.scss

```diff
@@ -1,384 +1,303 @@
-                      margin-bottom: rem(100);
-                      .no-results-title {
-                          h4 {
-                          margin-bottom: 0;
-                          vertical-align: bottom;
-                          }
-                      }
-                      .no-results-message {
-                          p {
-                          margin-top: 0;
-                          vertical-align: top;
-                          }
-                      }
-                      }
-                      .loading-animation {
-                          @import "../search/results/screens/loading";
-                      }
-                      .results-table-error {
-                      .icon {
-                          width: rem(36);
-                          height: rem(36);
-                          svg {
-                          width: rem(36);
-                          height: rem(36);
-                          }
-                      }
-                      .title {
-                          font-size: rem(20);
-                      line-height: rem(45);
-                      }
-                      .description {
-                          font-size: rem(16);
-                          line-height: rem(10);
-                      }
-                  }
-              }
-              .usda-table {
-                  td {
-                      &:first-child {
-                          @include span-columns(6 of 12, table);
-                          text-align: left;
-                      }
-                      @include span-columns(2 of 12, table);
-                      text-align: right;
-                      word-break: break-word;
-                  }
-
-                  .table-header {
-                      .table-header__label {
-                          @include flex(0 1 auto);
-                          text-align: right;
-                      }
-  
-                      .table-header__sort {
-                          margin-top: auto;
-                          margin-bottom: auto;
-                      }
-                  }
-              }
-          }
-          .body__section {
-              @import '../agency/v2/tabs';
-              @include display(flex);
-              @include flex-wrap(wrap);
-              margin-bottom: $global-mrg;
-              .body__header {
-                  width: 100%;
-                  @include display(flex);
-                  flex-wrap: wrap;
-                  align-items: center;
-                  justify-content: space-between;
-                  border-bottom: 3px solid $color-base;
-                  .body__header-left {
+    @import "all";
+    @import "layouts/default/default";
+    @import "layouts/default/stickyHeader/header";
+    @import "components/pageLoading";
+
+    @include display(flex);
+    @include justify-content(flex-start);
+    @include align-items(center);
+    @include flex-direction(column);
```

Comment:
Yes! This is always how i set up my containers... people hate on flex-direction(column) too much.